apps/passport-server/src/routing/routes/telegramRoutes.ts

```diff
@@ -275,4 +280,35 @@ export function initTelegramRoutes(
       res.status(500).send(errorHtmlWithDetails(e as Error));
     }
   });
+
+  app.get("/telegram/user", async (req: Request, res: Response) => {
```

Comment:
I would not call this a user in the url - it's something else. We already have something called user in the codebase, and I don't want the term to become overloaded.